index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+    <html>
```

Comment:
unindent everything below doctype

Reply:
I have unindented everything below the doctype as you requested.